virttest/qemu_storage.py

```diff
@@ -1550,10 +1550,49 @@ def measure(self, target_fmt, size=None, output="human"):
         cmd_list = [self.image_cmd, "measure", ("--output=%s" % output),
                     ("-O %s" % target_fmt)]
         if size:
+            if self._secret_objects:
+                cmd_list.extend(self._secret_objects)
+                secret_object_id = self.encryption_config.key_secret.aid
+                cmd_list.append("-o key-secret=%s" % secret_object_id)
             cmd_list.append(("--size %s" % size))
         else:
-            cmd_list.extend([("-f %s" % self.image_format),
-                             self.image_filename])
+            if self.encryption_config.key_secret:
+                image_filename = get_image_json(
+                        self.tag, self.params, self.root_dir)
+                image_filename = "'" + image_filename + "'"
+
+            source_secret_objects = self._secret_objects
+            target_secret_objects = []
+
+            if target_fmt == "luks":
+                convert_target = self.params["convert_target"]
```

Comment:
still, convert_target is used for convert only, can we introduce a new param? like   image_target or something else?

Reply:
As we discussed, set the parameter to image_target_image. Thanks.